dist/testbed-support/Makefile.iotlab

```diff
@@ -1,4 +1,4 @@
-.PHONY: iotlab-auth iotlab-exp iotlab-flash iotlab-reset iotlab-term
+.PHONY: iotlab-auth iotlab-exp iotlab-flash iotlab-reset iotlab-term check-exp
```

Comment:
please prefix `check-exp` with `iotlab-` or something else that is not so obvious